state/endpoint_bindings.go

```diff
@@ -545,7 +545,7 @@ func NewBindings(st EndpointBinding, givenMap map[string]string) (*Bindings, err
 func allOfOne(foundValue func(string) bool, givenMap map[string]string, allowEmptyValues bool) error {
 	for k, v := range givenMap {
 		if !foundValue(v) && (v != "" || (v == "" && !allowEmptyValues)) {
-			return errors.NotFoundf("endpoint %q, value %q, space name or id", k, v)
+			return errors.NotFoundf("space with name/id %q (endpoint %q)", v, k)
```

Comment:
Do we care about id here, I think it's probably better `space with name ...`

Reply:
We don't know necessarily if `givenMap` is a map with name-values or id-values. So we can't assume if v is a name or id.

In fact, this function is called on line 518 assuming that values are IDs, so will return this error for values that are names.

Perhaps a slash is the wrong conjunction here?